### PR TITLE
nasa-veda: added access to sentinel-cogs bucket

### PR DESCRIPTION
**Pull request by freitagb**

### CODE REVIEW

File: terraform/aws/projects/nasa-veda.tfvars [hunk 1]
```diff
@@ -63,7 +63,9 @@ hub_cloud_permissions = {
                     "arn:aws:s3:::sdap-dev-zarr",
                     "arn:aws:s3:::sdap-dev-zarr/*",
                     "arn:aws:s3:::usgs-landsat",
-                    "arn:aws:s3:::usgs-landsat/*"
+                    "arn:aws:s3:::usgs-landsat/*",
+                    "arn:aws:s3:::sentinel-cogs",
+                    "arn:aws:s3:::sentinel-cogs/*",
```
Comment (consideRatio) [inline terraform/aws/projects/nasa-veda.tfvars:68]:
```suggestion
                    "arn:aws:s3:::sentinel-cogs/*"
```

I don't think the trailing comma is accepted here, but I'm not sure.

Also, we should update this for the staging hub as well if i recall correctly.

---

I need to focus on misc other things for the moment and won't finalize getting this done.

Comment (freitagb) [reply terraform/aws/projects/nasa-veda.tfvars:68]: ahh yes. Thanks. I missed the trailing comma on the copy/paste. You can update for staging for consistency but it isn't needed at this time.

**Comment by sgibson91** [comment]
Output of terraform plan

```
Terraform used the selected providers to generate the following execution
plan. Resource actions are indicated with the following symbols:
  ~ update in-place

Terraform will perform the following actions:

  # aws_iam_policy.extra_user_policy["staging"] will be updated in-place
  ~ resource "aws_iam_policy" "extra_user_policy" {
        id          = "arn:aws:iam::444055461661:policy/nasa-veda-staging-extra-user-policy"
        name        = "nasa-veda-staging-extra-user-policy"
      ~ policy      = jsonencode(
          ~ {
              ~ Statement = [
                  ~ {
                      ~ Resource = [
                            # (27 unchanged elements hidden)
                            "arn:aws:s3:::usgs-landsat/*",
                          + "arn:aws:s3:::sentinel-cogs",
                          + "arn:aws:s3:::sentinel-cogs/*",
                        ]
                        # (2 unchanged attributes hidden)
                    },
                    {
                        Action   = "s3:ListAllMyBuckets"
                        Effect   = "Allow"
                        Resource = "*"
                    },
                ]
                # (1 unchanged attribute hidden)
            }
        )
        tags        = {}
        # (5 unchanged attributes hidden)
    }

Plan: 0 to add, 1 to change, 0 to destroy.
```

**Review by sgibson91 — APPROVED** [review]

**Comment by sgibson91** [comment]
Applied successfully

**Pull request merged**